mongo-powerbi-connector-downloads_template.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Des this need a content-type?

Reply:
Not that I know of. This is based on the file used by BIC and it's what the team from the download center reviewed and seemed OK with.
The other ones I checked like [this one](https://info-mongodb-com.s3.amazonaws.com/com-download-center/compass.json) seems to be the same too.